suites/src/main/java/com/hedera/block/suites/BaseSuite.java

```diff
@@ -47,6 +47,12 @@
  * <p>The Block Node Application version is retrieved dynamically from an environment file (.env).
  */
 public abstract class BaseSuite {
+    /**
+     * Dotenv instance to load the environment variables from the .env file that
+     * is inside the build root of the :server.
+     */
+    protected static final Dotenv DOTENV =
+            Dotenv.configure().directory("../server/build").filename(".env").load();
```

Comment:
This is not ideal, as it bypasses project isolation and ties this suite to a specific _other project_ build output.  Whatever runs _this suite_ should generate a _local_ `.env` and use that (if needed).  Otherwise the suite is tied to the server build via a file that's not part of the server outputs, and that's not a good pattern.

Reply:
Agreed, it is not ideal indeed. This was introduced as a temporary solution in the past. In theory, the test suites require the server docker image to be built, they are "coupled" in that sense, or rather the suites are coupled to the server, not the other way around. If we introduce a new `.env` file that is local to the suites, it will be an improvement, but yet again, the server version must still "come" from the server (granted there are other ways to handle that). I propose to make an issue to describe this and we will handle it in the future.

Reply:
If we have the suites build _depend_ on the server build, then the server artifacts (jar files, mostly) will be copied _by gradle_ into the build folder (I'd have to look up where, usually it's a "libs" folder).
We can (should?) get all of the server information from the jar if we've designed this correctly.

It's fine to create a separate issue to resolve this.

Reply:
#343 is now open and properly tagged within this PR's changes.